menustuff.json

```diff
@@ -2835,8 +2835,8 @@
   },
   "22301": {
     "OriginalText": "一時倉庫にアイテムが入っているため\nアクセスできません。\n一時倉庫を空にする必要があります。",
-    "Text": "",
-    "Enabled": false
+    "Text": "Because of items in Temp Storage,\naccess is denied.\nTemp Storage must be empty.",
```

Comment:
That should be fine. Generally being a bit verbose should be fine since the game can autofit things into the box as necessary, but there's a certain point where it starts to become unreadable, so it's kind of a case-by-case basis.